Packs/MicrosoftGraphMail/Integrations/MicrosoftGraphMail/MicrosoftGraphMail_test.py

```diff
@@ -444,3 +445,68 @@ def test_reply_mail_command(client, mocker):
     assert reply_message.outputs['subject'] == 'Re: ' + args['subject']
     assert reply_message.outputs['toRecipients'] == args['to']
     assert reply_message.outputs['bodyPreview'] == args['body']
+
+
+SEND_MAIL_COMMAND_ARGS = [
+    (
+        oproxy_client(),
+        {
+            'to': ['ex@example.com'],
+            'htmlBody': "<b>This text is bold</b>",
+            'subject': "test subject",
+            'from': "ex1@example.com"
+        },
+    ),
+    (
+        self_deployed_client(),
+        {
+            'to': ['ex@example.com'],
+            'htmlBody': "<b>This text is bold</b>",
+            'subject': "test subject",
+            'from': "ex1@example.com"
+        },
+    ),
+    (
+        oproxy_client(),
+        {
+            'to': ['ex@example.com'],
+            'body': "test body",
+            'subject': "test subject",
+            'from': "ex1@example.com"
+        }
+    ),
+    (
+        self_deployed_client(),
+        {
+            'to': ['ex@example.com'],
+            'body': "test body",
+            'subject': "test subject",
+            'from': "ex1@example.com"
+        }
+    )
+]
+
+
+@pytest.mark.parametrize('client, args', SEND_MAIL_COMMAND_ARGS)
+def test_send_mail_command(mocker, client, args):
+    """
+        Given:
+            - send-mail command's arguments
+
+        When:
+            - sending a mail
+
+        Then:
+            - validates that demisto result was called with the correct values.
+
+    """
+    mocker.patch.object(MicrosoftClient, 'http_request')
+    mocker.patch.object(demisto, 'results')
```

Comment:
better to use `requests_mock` to actually check the HTTP request is sent as expected

Reply:
not sure what you mean, i don't really care what would be returned from the http request in that test

Reply:
its not whats returned, but whats sent
we want to test that the html body is sent as expected

Reply:
ack, done